domains/webhooks.go

```diff
@@ -0,0 +1,66 @@
+package domains
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"time"
+)
+
+// Webhook represents webhook.
+type Webhook struct {
+	Url               string    `json:"url"`
+	TokenHeader       string    `json:"-"`
+	Token             string    `json:"-"`
+	CreatedAt         time.Time `json:"createdAt"`
+	LastEmitStatus    string    `json:"lastEmitStatus"`
+	LastEmitTimestamp time.Time `json:"lastEmitTimestamp"`
+	ErrorsCount       int       `json:"errorsCount"`
+	Active            bool      `json:"active"`
+}
+
+// Notify sends notification to webhook.
+func (w *Webhook) Notify(h *BlockHeader) (int, string, error) {
+	// Prepare the request.
+	headerBytes, err := json.Marshal(h)
+	if err != nil {
+		return 0, "", err
+	}
+	req, err := http.NewRequestWithContext(context.Background(), http.MethodPost, w.Url, bytes.NewReader(headerBytes))
+
+	if err != nil {
+		return 0, "", err
+	}
+
+	// Add the necessary headers.
+	req.Header.Add(w.TokenHeader, w.Token)
+	req.Header.Add("Content-Type", "application/json")
+
+	// Send the request.
+	client := &http.Client{}
+	res, err := client.Do(req)
```

Comment:
I have a doubt about using http in domain.
AFAIU in hexagonal arch we should probably pass here some component (implementation of the interface) that will handle the http request.
For example something like:
```golang
type WebhookTarget interface {
  Call(headers map[string][string], method http.MethodPost, url w.Url, body)
}
```

and implement it like here
and then pass such a component to Notify method.
So something like: 
`Notify(h *BlockHeader, target WebhookTarget)`

Reply:
In which case, I'm not sure WebhookTarget is the correct name. Webhook implies http so perhaps "WebTarget" is the interface name. Just a suggestion.

Reply:
@sirdeggen I was thinking about some interface to hide webhooks and websockets under it,
but I need to wait until this PR will be merged so it will be clear for me what and where should be "interfaced".